ignite/services/plugin/template/main.go.plush

```diff
@@ -41,7 +41,7 @@ func (p) Manifest(ctx context.Context) (*plugin.Manifest, error) {
 	}, nil
 }
 
-func (p) Execute(ctx context.Context, cmd *plugin.ExecutedCommand, az plugin.Analyzer) error {
+func (p) Execute(ctx context.Context, cmd *plugin.ExecutedCommand, c plugin.ClientAPI) error {
```

Comment:
Maybe we could call it `api` to set a standard name which is semantic?

```suggestion
func (p) Execute(ctx context.Context, cmd *plugin.ExecutedCommand, api plugin.ClientAPI) error {
```

Reply:
I mean for plugin developers. Calling it `api` would give a stringer semantic meaning when referencing it, like for example `ctx` for context or `cmd` for command.